yarn.lock

```diff
@@ -25738,16 +25182,16 @@ typescript-json-schema@^0.50.1:
     typescript "~4.2.3"
     yargs "^16.2.0"
 
-typescript@^4.0.3, typescript@~4.1.3:
-  version "4.2.3"
-  resolved "https://registry.npmjs.org/typescript/-/typescript-4.2.3.tgz#39062d8019912d43726298f09493d598048c1ce3"
-  integrity sha512-qOcYwxaByStAWrBf4x0fibwZvMRG+r4cQoTjbPtUlrWjBHbmCAww1i448U0GJ+3cNNEtebDteo/cHOR3xJ4wEw==
-
-typescript@~4.2.3:
+typescript@^4.0.3, typescript@~4.2.3:
   version "4.2.4"
   resolved "https://registry.npmjs.org/typescript/-/typescript-4.2.4.tgz#8610b59747de028fda898a8aef0e103f156d0961"
   integrity sha512-V+evlYHZnQkaz8TRBuxTA92yZBPotr5H+WhQ7bD3hZUndx5tGOa1fuCgeSjxAzM1RiN5IzvadIXTVefuuwZCRg==
 
+typescript@~4.1.3:
+  version "4.2.3"
```

Comment:
Hmm, this one doesn't seem right 🤔  `~4.1.3` shouldn't go to `4.2.x`, right?

Reply:
The tilde is an odd operator. It will allow minor changes most of the time to only one minor version change, so technically according to [this](https://docs.npmjs.com/cli/v7/using-npm/semver/#tilde-ranges-123-12-1), the change does meet the semver